Assets/BossRoom/Scripts/Client/UI/PostGameUI.cs

```diff
@@ -76,7 +77,7 @@ void SetPostGameUI(WinState winState)
         public void OnPlayAgainClicked()
         {
             // this should only ever be called by the Host - so just go ahead and switch scenes
-            NetworkManager.Singleton.SceneManager.LoadScene("CharSelect", LoadSceneMode.Single);
+            SceneLoader.Instance.LoadScene("CharSelect", LoadSceneMode.Single);
```

Comment:
shouldn't the switch to CharSelect remain synced with the other clients and the host?